frontend/src/metabase/dashboard/components/TextOptions/TextChoicesPopover.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
```

Comment:
We have a menu like this in other places like the "New" button, wdyt about reusing it?
<img width="232" alt="Screen Shot 2023-05-25 at 9 12 07 PM" src="https://github.com/metabase/metabase/assets/14301985/203963e1-140e-4c1e-9cc0-153c214910ef">

Reply:
Thanks for pointing this out!
I will test this out and get designs feedback for it.